gradle.properties

```diff
@@ -1,27 +1,11 @@
-# =============== Kotlin Gradle plugin configuration ===============
-kotlin_version=1.3.50
-
-# change the variable value to download a specific build from TeamCity
-# currently broken: https://youtrack.jetbrains.com/issue/KT-34246
-custom_build_id=2478816
-
-# =============== Other libraries versions ===============
-
-#coroutines_version=1.3.2-1.3.60-eap-76
-coroutines_version=1.3.2
-
-#serialization_version=0.14.0-1.3.60-eap-76
-serialization_version=0.13.0
-android_device_names_version=1.1.9
-ktor_version=1.2.5
```

Comment:
I'd rather leave it as is. I find it the way more convenient to have all the versions in one place and be able to change them with a new Kotlin version release with the corresponding libraries update.

Reply:
How about...[this](https://github.com/wild-lynx/sample-mpp-android-app/pull/2/files/b46cc3af492c2ee8b871f4d5054bdcd013cbd77b..b6aae49d2a4aaf38f98369e0f698789de338df3a#diff-6fd73db99d0ec5c6891d940c91891b2dR3-R37)? 😄